config/database.ts

```diff
@@ -0,0 +1,19 @@
+import mongoose from 'mongoose';
+
+export async function connect(): Promise<void> {
+    const { MONGO_DEV_URI } = process.env;
+
+    if (!MONGO_DEV_URI) {
+        console.log("Please provide DataBase URI to connect. exiting now...");
+        process.exit(1);
```

Comment:
Consider throwing an error instead of calling `process.exit(1)` directly

Reply:
It was taken from example code in learning materials.
Thanks for your advise.